addons/im_livechat/static/tests/sidebar_patch_tests.js

```diff
@@ -9,11 +9,11 @@ QUnit.module("discuss sidebar (patch)");
 
 QUnit.test("Unknown visitor", async (assert) => {
     const pyEnv = await startServer();
+    const guestId = pyEnv["mail.guest"].create({ name: "Visitor 11" });
     pyEnv["discuss.channel"].create({
-        anonymous_name: "Visitor 11",
         channel_member_ids: [
             [0, 0, { partner_id: pyEnv.currentPartnerId }],
-            [0, 0, { partner_id: pyEnv.publicPartnerId }],
+            [0, 0, { guest_id: guestId }],
```

Comment:
one of them was changed to command create, maybe change all of the line with diff in this pr